adr/ADR-43.md

```diff
@@ -70,3 +79,8 @@ type StreamConfig struct {
 	LimitsTTL   time.Duration `json:"limits_ttl"`
```

Comment:
Following discussion we have agreed on:
```
SubjectDeleteMarkers   bool   `json:"subject_delete_markers,omitempty"`
SubjectDeleteMarkerTTL string `json:"subject_delete_marker_ttl,omitempty"`
```
... the latter being a string because the `Nats-TTL` header can either be numeric or a duration.

Default TTL will be 15 minutes, as it's long enough to survive just about any outage we can think of (including ordered consumer recreates, client timeouts/reconnects, route/gateway timeouts, etc).